build.sh

```diff
@@ -39,6 +39,7 @@ HELP="$0 [<target> ...] [<flag> ...]
    -n               - no install step
    --allgpuarch     - build for all supported GPU architectures
    --buildfaiss     - build faiss statically into libcuml
+   --buildgtest     - build googltest library
```

Comment:
FYI cudf uses `tests` as the flag for building all tests. I know this is specific to google tests, but worth considering keeping flags consistent

https://github.com/rapidsai/cudf/blob/9e972cefe561131636dae584940ec16c2bb1a1e3/build.sh#L28

Reply:
Also `googltest` should be `googletest`

Reply:
I think the intent of the cudf `tests` flag and this flag is different, this one in particular is not to build libcuml tests but to build googletest as part of the cmake build process as opposed to using the conda package, so I think it would be more consistent to keep the different name (correct me if I misunderstood the intent of the cuDF `tests` flag)